projects/frontend/cicd/build_data_pipelines.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+###
+# For installing the build prerequisites, consider `install_data_pipelines.sh` script.
+# This script is meant to be used, to quickly
+# rebuild and link the the @vdk/data-pipelines in an already bootstrapped development setup.
+###
+
+if ! which npm >/dev/null 2>&1 ; then
+  echo "ERROR:"
+  echo "Please install npm 8.5.5+. Build cannot continue without it."
+  exit 1
+fi
+echo "Logging npm engines version..."
+npm version
+
+cd "$(dirname $0)" || exit 1
+cd "../data-pipelines/gui"
```

Comment:
What's the point of executing this and the previous line?

Reply:
L20 so that the script navigates on FS correctly, independently of where it is executed from
L21 is to navigate, so that other steps are relative to this one (data-pipelines component source root)

Reply:
It's a standard practice, we use it a lot in VDK codebase

Reply:
Makes sense when you take into account local reusability.